### PR TITLE
On demand environment: adjust for new environment

### PR DESCRIPTION
**Pull request by chmanie**
## Description

This PR adjusts the on-demand environment to all the things we've changed in between for production and makes it actually, truly work this time :)

## BREAKING:

You HAVE to set the `REPUTATION_ORACLE_ENDPOINT` variable in your `.env` file now. I did this to make it flexible and to make it consistent with the other variables.

### CODE REVIEW

**Review by TomaszChmielPagepro — APPROVED** [review]

File: scripts/create-data.js [hunk 1]
```diff
@@ -1354,10 +1485,19 @@ const checkNodeVersion = () => {
 };
 
 const pressKeyToContinue = async () => {
+  if (yesArg > -1) {
```
Comment (chmanie) [inline scripts/create-data.js:1488]: Here's another actual change that I made

File: scripts/create-data.js [hunk 2]
```diff
@@ -39,11 +44,12 @@ const {
   users: usersTempData,
 } = require('./tempColonyData');
 
-
 // fetch command line arguments
 const timeoutArg = process.argv.indexOf('--timeout');
 const timeoutArgValue = process.argv[timeoutArg + 1];
 
+const yesArg = process.argv.indexOf('--yes');
```
Comment (chmanie) [inline scripts/create-data.js:51]: Here's the first actual change that I made

File: scripts/create-data.js
Comment (chmanie) [inline scripts/create-data.js:1]: I formatted this file to adhere to our code linter rules (this was gonna happen eventually)

**Review by bogdan-1337 — APPROVED** [review]

**Review by TomaszChmielPagepro — APPROVED** [review]

**Comment by area** [comment]
Interested if setting `REPUTATION_ORACLE_ENDPOINT` and `GANACHE_ACCOUNTS_ENDPOINT` in the `.env` file is enough for @arrenv's devstack - I believe it should be. Otherwise, yeah, happy to take this in to my PR.

**Pull request merged**